lib/benchmark/benchmark.go

```diff
@@ -0,0 +1,127 @@
+package benchmark
```

Comment:
nit: either merge `bench.go` and `benchmark.go` or rename one of them to match the contents.